.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,7 +26,7 @@ Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-
   - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
   - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
   - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
-- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
+- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review pr PRNUMBER"`. see [usage](https://github.com/Mic92/nixpkgs-review#usage)
```

Comment:
One problem with this is that it's in a checklist you fill out when opening a PR, so you don't actually have a PR number to run it with. Similar thing with `rev HEAD` because it doesn't include uncommitted changes. Maybe use `rev HEAD` but explain all changes need to be committed?

Reply:
I would also prefer having `rev HEAD` instead